tutorials/platform/android/android_plugin.rst

```diff
@@ -49,8 +49,12 @@ with the following caveats:
 Building an Android plugin
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-**Prerequisite:** `Android Studio <https://developer.android.com/studio>`_ is strongly recommended as the IDE to use to create Android plugins.
-The instructions below assumes that you're using Android Studio.
+**Prerequisite:** Android Studio is strongly recommended as the IDE to use to create Android plugins. Currently the latest version of Android Studio
+is not compatible with Godot because it requires JDK 17 and Godot requires JDK 11.
+(see release notes `here <https://developer.android.com/build/releases/gradle-plugin#jdk-17-agp>`_ and `here <https://developer.android.com/build/releases/gradle-plugin#8-0-0>`_).
```

Comment:
The build fails because of: `android_plugin.rst:6:Duplicate explicit target name: "here"`
You should not use duplicate names for links. Instead try to use proper self contained names.

```suggestion
(see the release notes of `JDK 17 <https://developer.android.com/build/releases/gradle-plugin#jdk-17-agp>`_ and `AGP 8.0 <https://developer.android.com/build/releases/gradle-plugin#8-0-0>`_).
```